### PR TITLE
Move kubernetes-kubemark-scale-gce Jenkins job to asia-east1-a zone.

### PR DESCRIPTION
**Pull request by ghost**
Because that's where we have enough quota.

### CODE REVIEW

**Comment by quinton-hoole** [comment]
Self-merging to unbreak kubernetes-kubemark-scale-gce Jenkins job.
cc: @gmarek @wojtek-t

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -650,6 +650,7 @@ case ${JOB_NAME} in
     : ${E2E_TEST:="false"}
     : ${USE_KUBEMARK:="true"}
     : ${NUM_MINIONS:="40"}
+    : ${E2E_ZONE:="asia-east1-a"}
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: This isn't going to work. `E2E_ZONE` is already defined on L71, so this won't override.

Comment (wojtek-t) [inline hack/jenkins/e2e.sh]: yup - we should overwrite similarly to what we do for scalability (~ L350).

@gmarek - can you please fix it?

**Comment by gmarek** [comment]
Yes I can, but I'm not sure it's worth doing 4k experiments. I'll bring it down to a 1000 node one. More on it in #15064

**Pull request merged**